packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -100,6 +100,14 @@ const getGatsbyImageCdnFields = async ({
 
     return gatsbyImageCdnFields
   } catch (e) {
+    if (e.message.includes(`unrecognized file format`)) {
+      reporter.error(
+        `Encountered corrupt file while requesting image dimensions for ${url}`
```

Comment:
Add a prefix saying this is from source-drupal as well?

Reply:
Good call!